src/python/grpcio_tests/tests/interop/methods.py

```diff
@@ -44,25 +44,48 @@
 from src.proto.grpc.testing import messages_pb2
 from src.proto.grpc.testing import test_pb2
 
+_INITIAL_METADATA_KEY = "x-grpc-test-echo-initial"
+_TRAILING_METADATA_KEY = "x-grpc-test-echo-trailing-bin"
+
+# If the request came with metadata in the context, this
+# helper function will copy it to the returned metadata.
+def MaybeEchoMetaData(context):
```

Comment:
The lack of leading underscore on this identifier indicates that you are making it module-public (i.e. you expected it to be used outside this module and support such usage). If that's not the case, please append a leading underscore to it. And then do the same with every other module-scope code element introduced in this change.